4.figures/supp_figure_4/scripts/SuppFigure4_platemaps.r

```diff
@@ -4,11 +4,11 @@ suppressPackageStartupMessages(library(platetools))
 suppressPackageStartupMessages(library(patchwork))
 
 figure_dir <- "../figures/supplementary"
-output_supp_figure_2 <- file.path(figure_dir, "supp_figure_2_platemaps.png")
+output_supp_figure <- file.path(figure_dir, "supp_figure_4_platemaps.png")
```

Comment:
The platemaps should indicate `NF1` somewhere - probably `NF1 genotype` 🤷 

Also, would it be appropriate to show the cell line code here somewhere? (the answer might easily be no, but then again, should we include plate 6 platemap?)

Reply:
We include that Plate 6 platemap as a part of Figure 6 Panel A so we don't need to include it in this supplemental. I am happy to rename to include reference to NF1 genotype. Also the platemaps do include the cell line code under "NF1 genotype" :)